pacehrh/vignettes/pacehrh.rmd

```diff
@@ -260,3 +261,10 @@ sheets and run via an R script.
 
     The results are saved out to a CSV file with the date and name of the geography as specified
     in Excel in the filename. The console will beep at you when the model has finished running. 
+    
+### Install latest version via Github
```

Comment:
I think we need a different title for this section, something that makes it clear that by "latest" you mean "bleeding edge, not officially released" code.